### PR TITLE
basic functional answer board done

### PR DESCRIPTION
**Pull request by mountwebs**

### CODE REVIEW

File: src/components/Input/Input.js [hunk 1]
```diff
@@ -37,20 +37,52 @@ const StyledContainer = styled.div`
...
+    answer.tema = temaValue;
+    addAnswer(answer);
+  };
+
+  const answer = {
+    text:
+      "Minst like viktig å ruste opp og se nye muligheter i det som allerede finnes, som å nødvendigvis måtte bygge alt fra scratch. ",
+    tema: "Gjenbruk",
+    user: "Navn",
+  };
```
Comment (carlos-ch) [inline src/components/Input/Input.js:64]: could declare the answer object directly in the handleSubmit, without having to change an existing one

File: src/components/Input/Input.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
-import React from 'react';
-import styled from 'styled-components';
-import Button from '../Button/Button';
+import React, { useState } from "react";
+import styled from "styled-components";
+import Button from "../Button/Button";
 
-const StyledContainer = styled.div`
+const StyledContainer = styled.form`
```
Comment (carlos-ch) [inline src/components/Input/Input.js:5]: This was definitely a better tag!:D

**Review by carlos-ch — APPROVED** [review]
🙆‍♂️ very nice

File: src/components/Input/Input.js (hunk 1 above)
Comment (mountwebs) [reply src/components/Input/Input.js]: Done:)

**Pull request merged**